src/GitHub/Data/Teams.hs

```diff
@@ -165,19 +165,21 @@ instance FromJSON Team where
         <*> o .: "organization"
 
 instance ToJSON CreateTeam where
-  toJSON (CreateTeam name desc repo_names {-privacy-} permissions) =
+  toJSON (CreateTeam name desc repo_names privacy permission) =
     object [ "name"        .= name
            , "description" .= desc
            , "repo_names"  .= repo_names
-           {-, "privacy" .= privacy-}
-           , "permissions" .= permissions ]
+           , "privacy"     .= privacy
+           , "permission"  .= permission
+           ]
 
 instance ToJSON EditTeam where
-  toJSON (EditTeam name desc {-privacy-} permissions) =
+  toJSON (EditTeam name desc privacy permission) =
     object [ "name"        .= name
            , "description" .= desc
-           {-, "privacy" .= privacy-}
-           , "permissions" .= permissions ]
+           , "privacy"     .= privacy
```

Comment:
now as `privacy` is `Maybe` we should encode it differently so no `null` is passed. (current code will pass it); look around there's code which makes an object with filtered out nulls.